packages/flutter_tools/test/integration.shard/test_data/multidex_project.dart

```diff
@@ -162,7 +162,7 @@ class MultidexProject extends Project {
   apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
   android {
-      compileSdkVersion 30
+      compileSdkVersion 31
```

Comment:
was there an issue with the current compileSdkVersion?

Reply:
Yeah, this test is looking for a specific failure, and without bumping this version, it failed my test before giving the failure expected

Reply:
cool, but the test was still passing, right? Your PR is mostly adding more errors to the stderr

Reply:
No, the test was failing. @GaryQian can you provide some more context here?